network_wrangler/models/roadway/tables.py

```diff
@@ -35,24 +35,23 @@ class RoadLinksTable(DataFrameModel):
     A: Series[int] = pa.Field(nullable=False, coerce=True)
     B: Series[int] = pa.Field(nullable=False, coerce=True)
     geometry: GeoSeries = pa.Field(nullable=False)
-    name: Series[str] = pa.Field(nullable=False)
+    name: Series[str] = pa.Field(nullable=True)
     rail_only: Series[bool] = pa.Field(coerce=True, nullable=False, default=False)
     bus_only: Series[bool] = pa.Field(coerce=True, nullable=False, default=False)
     drive_access: Series[bool] = pa.Field(coerce=True, nullable=False, default=True)
     bike_access: Series[bool] = pa.Field(coerce=True, nullable=False, default=True)
     walk_access: Series[bool] = pa.Field(coerce=True, nullable=False, default=True)
-    distance: Series[float] = pa.Field(coerce=True, nullable=False)
+    distance: Series[float] = pa.Field(coerce=True, nullable=True)
 
     roadway: Series[str] = pa.Field(nullable=False, default="road")
     managed: Series[int] = pa.Field(coerce=True, nullable=False, default=0)
 
     shape_id: Series[str] = pa.Field(coerce=True, nullable=True)
-    lanes: Series[int] = pa.Field(coerce=True, nullable=False)
+    lanes: Series[Any] = pa.Field(coerce=True, nullable=True, default=0)
     price: Series[float] = pa.Field(coerce=True, nullable=False, default=0)
```

Comment:
lanes variable has mix data types

Reply:
Not after being read-in, it shouldn't.

Reply:
fixed by dropping this column in input json file